_posts/2017-09-18-a-day-in-the-life-of-an-18f-product-owner.md

```diff
@@ -30,7 +30,7 @@ We interviewed our product owner, Aaron Burk, for our Forest Service [engagement
 
 ### TTS: Can you walk me through what you do as a product owner? 
 
-**Aaron Burk:** As a product owner, my role is to ensure that our partners at GSA/TTS/18F have the information necessary to put together the appropriate solicitation documents as we start to engage contractors selected from the [Agile Blanket Purchase Agreement](). As we work through that process, the product owner needs to be able to weigh in and help evaluate the results of the bids we receive in relation to the work that we need done, as we have for our first two buys, the logic layer [Application Program Interface](https://github.com/18F/fs-middlelayer-api) and the [intake module](https://github.com/18F/fs-intake-module). Once we select a vendor, my role is a continued relationship with the vendor, alongside TTS, to keep the communications open to get the product that the Forest Service requires to best serve the public’s needs. I have wonderful resources from the TTS team to help with the technical aspects of the project. It’s key to leverage the resources we have but also strongly communicate the values of the agency and the users of the system.
+**Aaron Burk:** As a product owner, my role is to ensure that our partners at GSA/TTS/18F have the information necessary to put together the appropriate solicitation documents as we start to engage contractors selected from the Agile Blanket Purchase Agreement. As we work through that process, the product owner needs to be able to weigh in and help evaluate the results of the bids we receive in relation to the work that we need done, as we have for our first two buys, the logic layer [Application Program Interface](https://github.com/18F/fs-middlelayer-api) and the [intake module](https://github.com/18F/fs-intake-module). Once we select a vendor, my role is a continued relationship with the vendor, alongside TTS, to keep the communications open to get the product that the Forest Service requires to best serve the public’s needs. I have wonderful resources from the TTS team to help with the technical aspects of the project. It’s key to leverage the resources we have but also strongly communicate the values of the agency and the users of the system.
```

Comment:
Removed a link to the Agile Blanket Purchase Agreement. Apparently we've removed all references to it from our site, except for blog posts. Maybe we should link to [this blog post](https://18f.gsa.gov/2018/07/26/what-we-learned-from-building-a-pool-of-agile-vendors/) where we kind of do a retro on the Agile BPA after we sunset it?